### PR TITLE
Aditya

### PR DESCRIPTION
**Pull request by adityajatiya**

### CODE REVIEW

File: src/javapractice/Basic/DataType/CharacterDataType.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+import java.util.Scanner;
+
+public class CharacterDataType {
+    
+    public static void main(String[] args) {
+        System.out.print(" Your Grade = ");
+        char grade= 'a';
+          System.out.println(grade);
```
Comment (Madhur-Jatiya) [inline src/javapractice/Basic/DataType/CharacterDataType.java]: format code remove space before System.out

File: src/javapractice/Basic/DataType/DoubleDataType.java [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.print("Enter Your 10th Class Percentage = ");
+//        String d = sc.next();
+//        double x = sc.nextDouble();
+//        System.out.printf("%.13f%n", x);
+
+        
+        double s = 82174932843.3424632764732d;
```
Comment (Madhur-Jatiya) [inline src/javapractice/Basic/DataType/DoubleDataType.java]: take input from user

File: src/javapractice/Basic/DataType/LongDataType.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package javapractice.Basic.DataType;
+
+import java.util.Scanner;
+
+public class LongDataType {
+    public static void main(String []args){
+        Scanner sc = new Scanner(System.in);
+         System.out.println("Enter Long(Max. Size 20) Number =");
+        long x = sc.nextLong();
```
Comment (Madhur-Jatiya) [inline src/javapractice/Basic/DataType/LongDataType.java]: format code

File: src/javapractice/Basic/String/StringByNewKeyword.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Name Enter = ");
+        String name = new String();
+        name = sc.nextLine();
+
+        System.out.print("Name = " + name + "\n");
+
+        name = "ram";
+        System.out.print("Name = " + name + "\n");
```
Comment (Madhur-Jatiya) [inline src/javapractice/Basic/String/StringByNewKeyword.java]: remove these lines

File: src/javapractice/Basic/String/StringLiteral.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Name Enter = ");
+        String name = sc.nextLine();
+
+        System.out.print("Name = " + name + "\n");
+
+        name = "ram";
+        System.out.print("Name = " + name + "\n");
```
Comment (Madhur-Jatiya) [inline src/javapractice/Basic/String/StringLiteral.java]: remove these lines

File: src/javapractice/Basic/String/StringByNewKeyword.java (hunk 4 above)
Comment (adityajatiya) [reply src/javapractice/Basic/String/StringByNewKeyword.java]: fix

File: src/javapractice/Basic/DataType/LongDataType.java (hunk 3 above)
Comment (adityajatiya) [reply src/javapractice/Basic/DataType/LongDataType.java]: fix

File: src/javapractice/Basic/DataType/DoubleDataType.java (hunk 2 above)
Comment (adityajatiya) [reply src/javapractice/Basic/DataType/DoubleDataType.java]: fix

File: src/javapractice/Basic/DataType/CharacterDataType.java (hunk 1 above)
Comment (adityajatiya) [reply src/javapractice/Basic/DataType/CharacterDataType.java]: fix

File: src/javapractice/Basic/String/StringLiteral.java (hunk 5 above)
Comment (adityajatiya) [reply src/javapractice/Basic/String/StringLiteral.java]: fix

**Comment by adityajatiya** [comment]
fix problems

**Pull request closed without merging**